testing/tests/DevExpress.ui.events/eventsEngine.tests.js

```diff
@@ -245,6 +245,25 @@ QUnit.test("Event bubbling", function(assert) {
     assert.equal(fired.focus, 0);
 });
 
+QUnit.test("Should not fire event when relatedTarget is children of a target", function(assert) {
+    var div = document.createElement("div"),
+        div2 = document.createElement("div"),
+        fired = 0;
+    div.appendChild(div2);
+
+    document.body.appendChild(div);
+
+    eventsEngine.on(div, "someEvent", function() {
+        fired++;
+    });
+
+    var event = new eventsEngine.Event("someEvent", { target: div, relatedTarget: div.children[0] });
```

Comment:
It's better to rename `div2` to something like `childNode` and use it instead of ` div.children[0]`.